AI_VoiceOver/Modules/Options/GeneralTab/VoiceOverOptionsGeneral.lua

```diff
@@ -0,0 +1,177 @@
+								Addon.soundQueue.ui.db["ShowFrameBackground"] = value
+								Addon.soundQueue.ui:refreshConfig()
+							end,
+						},
+						hideNpcHead = {
+							type = "toggle",
+							order = 4,
+							name = function() return "Hide NPC Head"; end,
+							desc = function() return "Talking NPC head will not appear when voice over audio is played.\n\n" ..
+								"(This might be useful when using other addons that replace the dialog experience, such as " ..
+								VoiceOverUtils:colorizeText("Immersion", VoiceOverUtils.ColorCodes.DefaultGold); end,
+							get = function () return Addon.soundQueue.ui.db["HideNpcHead"]; end,
+							set = function(info, value)
+								Addon.soundQueue.ui.db["HideNpcHead"] = value
+								Addon.soundQueue.ui:refreshConfig()
+								Addon.soundQueue.ui:updateSoundQueueDisplay()
+							end,
+						},
+					},
+				},
+				audio = {
+					type = "group",
+					order = 4,
+					inline = true,
+					name = function() return "Audio"; end,
+					args = {
+						soundChannel = {
+							type = "select",
+							width = 0.75,
+							order = 1,
+							name = function() return "Sound Channel"; end,
+							desc = function() return "Controls which sound channel VoiceOver will play in."; end,
+							values = {
+								["Master"] = "Master",
+								["Sound"] = "Sound",
+								["Ambience"] = "Ambience",
+								["Music"] = "Music",
+								["Dialog"] = "Dialog",
+							},
+							get = function() return Addon.soundQueue.ui.db["SoundChannel"]; end,
+							set = function(info, value)
+									Addon.soundQueue.ui.db["SoundChannel"] = value
+									Addon.soundQueue.ui:refreshConfig()
+							end,
+						},
+						gm_spacer = {
+							type = "description",
+							order = 2,
+							width = 0.1,
+							name = function() return " "; end
+						},
+						gossipPlaybackFrequency = {
+							type = "select",
+							width = 1.25,
+							order = 3,
+							name = function() return "Gossip Playback Frequency"; end,
+							desc = function() return "Controls how often VoiceOver will play dialog."; end,
+							values = {
+								["Always"] = "Always",
+								["OncePerQuestNpc"] = "Play Once for Quest NPCs",
+								["OncePerNpc"] = "Play Once for All NPCs",
+								["Never"] = "Never",
+							},
+							get = function() return Addon.soundQueue.ui.db["GossipFrequency"]; end,
+							set = function(info, value)
+									Addon.soundQueue.ui.db["GossipFrequency"] = value
+									Addon.soundQueue.ui:refreshConfig()
+							end,
+						},
+						muteWhileVoiceOverPlaying = {
+							type = "toggle",
+							width = 1.75,
+							order = 4,
+							name = function() return "Mute NPCs While VoiceOver is Playing"; end,
+							desc = function() return "While VoiceOver is playing, the Dialog channel will be muted."; end,
+							get = function () return Addon.soundQueue.ui.db["AutoToggleDialog"]; end,
+							set = function(info, value)
+									Addon.soundQueue.ui.db["AutoToggleDialog"] = value
```

Comment:
Addon.soundQueue.ui.db["AutoToggleDialog"] and Addon.db.profile.main.AutoToggleDialog point to the same thing. Lets stick to using Addon.db instead of soundQueue.ui.db